va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -641,6 +655,7 @@ def update_options(search_value, location_json):
     [
         Output(component_id="filter_dict", component_property="children"),
         Output(component_id="timeframe", component_property="disabled"),
+#        Output(component_id="bounds", component_property="children")
```

Comment:
Do we see this commented out line as being useful going forward? If not, I'd recommend removal.